src/Mod/Spreadsheet/Gui/DlgSettingsImp.cpp

```diff
@@ -0,0 +1,115 @@
+/***************************************************************************
+ *   Copyright (c) 2002 Jürgen Riegel <juergen.riegel@web.de>              *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+
+#include "PreCompiled.h"
+#ifndef _PreComp_
+# include <QApplication>
+#endif
+
+#include "DlgSettingsImp.h"
+#include "ui_DlgSettings.h"
+#include <Gui/Application.h>
+#include <Gui/PrefWidgets.h>
+#include <Base/Console.h>
+
+using namespace SpreadsheetGui;
+
+/* TRANSLATOR SpreadsheetGui::DlgSettingsImp */
+
+/**
+ *  Constructs a DlgSettingsImp which is a child of 'parent', with the
+ *  name 'name' and widget flags set to 'f' 

```

Comment:
I know it's copy-and-paste boilerplate... but it's still wrong, there is no "name" or "f" here.